clusterloader2/testing/load/config.yaml

```diff
@@ -48,12 +48,38 @@
 {{$mediumDeploymentsPerNamespace := SubtractInt $mediumDeploymentsPerNamespace (IfThenElse $ENABLE_JOBS 1 0)}}
 {{$bigDeploymentsPerNamespace := SubtractInt $bigDeploymentsPerNamespace (IfThenElse $ENABLE_JOBS 1 0)}}
 
+# The minimal number of pods to be used to measure various things like
+# pod-startup-latency or scheduler-throughput. The purpose of it is to avoid
+# problems in small clusters where we wouldn't have enough samples (pods) to
+# measure things accurately.
+{{$MIN_PODS_IN_SMALL_CLUSTERS := 500}}
+
+# BEGIN scheduler-throughput section
+{{$totalSchedulerThroughputPods := MaxInt $MIN_PODS_IN_SMALL_CLUSTERS (MultiplyInt 2 .Nodes)}}
+{{$schedulerThroughputPodsPerDeployment := .Nodes}}
+{{$schedulerThroughputNamespaces := DivideInt $totalSchedulerThroughputPods $schedulerThroughputPodsPerDeployment}}
+{{$schedulerThroughputThreshold := DefaultParam .CL2_SCHEDULER_THROUGHPUT_THRESHOLD 0}}
+# END scheduler-throughput section
+
+# TODO(https://github.com/kubernetes/perf-tests/issues/1024): Ideally, we wouldn't need this section.
```

Comment:
nit:
s/Ideally .../Investigate and get rid of this section./

Reply:
Done.